templates/konvoy-ui.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: konvoy-ui
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: konvoy-ui
+  annotations:
+    appversion.kubeaddons.mesosphere.io/kommander: "1.40.0"
+    endpoint.kubeaddons.mesosphere.io/landing: /landing
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: kommander
+    repo: https://mesosphere.github.io/charts/stable
+    version: 0.1.6
+    values: |
```

Comment:
I just noticed we need the environment variable `MODE=konvoy` in the kommander chart. I would change the chart to allow the user to pass down environment variables through the values.